HLTrigger/Timer/plugins/FastTimerService.cc

```diff
@@ -767,7 +767,7 @@ void FastTimerService::PlotsPerJob::fill_lumi(AtomicResources const& data, unsig
 ///////////////////////////////////////////////////////////////////////////////
 
 FastTimerService::FastTimerService(const edm::ParameterSet& config, edm::ActivityRegistry& registry)
-    : tbb::task_scheduler_observer(true),
```

Comment:
this was introduced in #33261; I don't know what the change does, but maybe @dan131riley can comment ?

Reply:
I think this was needed to prevent the coroutine from being deleted before this service ended. The early deletion caused a segfault.

Reply:
@dan131riley can you confirm.

Reply:
See https://github.com/cms-sw/cmssw/blob/6d2f66057131baacc2fcbdd203588c41c885b42c/FWCore/Services/plugins/InitRootHandlers.cc#L75

Reply:
onetbb_2021 removed the constructor that takes a global/local boolean.  While the comments and docs still mention global observers, that actually isn't an option anymore--the only. constructors are a default (that actually does nothing) and one that takes a `task_arena&` to observe a specific arena other than the current one.  See

https://github.com/oneapi-src/oneTBB/blob/2dba2072869a189b9fdab3ffa431d3ea49059a19/include/oneapi/tbb/task_scheduler_observer.h#L74-L85
https://spec.oneapi.com/versions/latest/elements/oneTBB/source/task_scheduler/task_arena/task_scheduler_observer_cls.html

In fact, we could (and probably should) just remove that line.

Reply:
So `public tbb::task_scheduler_observer` should be removed in both locations?

Reply:
`ThreadTracker` and `FastTimerService`?

Reply:
The member variable must stay, but the explicit call to the default constructor can be removed.

Reply:
It's actually a parent class, and indeed the parent class needs to stay, it's just the explicit call to the constructor, the `tbb::task_scheduler_observer()`,  that can be removed, since the default constructor will be called anyway.